internal/interactive/prompt.go

```diff
@@ -153,7 +153,7 @@ func Readline(config *ReadlineConfig) (string, error) {
 			case "Interrupt":
 				return "", &InterruptError{}
```

Comment:
I think it would be better to return a CliError with an exit code here

Reply:
are we sure we want to return a `CliError` from `interactive` ?